azuredisk-csi.yaml

```diff
@@ -0,0 +1,71 @@
+package:
+  name: azuredisk-csi
+  version: 1.31.0
+  epoch: 0
+  description: Azure Disk CSI Driver
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  environment:
+    CGO_ENABLED: "0"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 81f73ae6e758c2dd5efbaf127014e80d01d39f66
+      repository: https://github.com/kubernetes-sigs/azuredisk-csi-driver
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      ldflags: |
+        -X sigs.k8s.io/azuredisk-csi-driver/pkg/azuredisk.driverVersion=v${{package.version}}
+        -X sigs.k8s.io/azuredisk-csi-driver/pkg/azuredisk.gitCommit=$(git rev-parse HEAD)
+        -X sigs.k8s.io/azuredisk-csi-driver/pkg/azuredisk.buildDate=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+        -extldflags "-static"
```

Comment:
> -extldfags -static passes -static to ld linker, which in practice means the same thing as CGO_ENABLED=0 which is what we want to avoid during FIPS package builds for Go projects

from wiki, this is non-fips but we can consider omitting here too.

Reply:
TIL, thanks Anurag, will ommit it